rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -200,19 +200,7 @@ then
   RESTORE_SERVER_ROLES_BACKUP=1
 fi
 
-# Copy initial promises if there aren't any already or,
```

Comment:
Don't we need another fallback? (like remove policies if they are corrupted?)

Reply:
a reset ?